supported-smartphones.md

```diff
@@ -1,72 +1,100 @@
 
-### Bluetooth 5 Long Range Support - the Ability to Receive Data
+Both of the rows must say YES, in order for the device to be able to receive the Remote ID Bluetooth Long Range broadcast signals.
 
-Suppose the previous test did not eliminate the device from Long Range support. In that case, it is now necessary to test the target device’s basic receiving capability. At the moment, this information cannot be read from the OS information and must be verified by using another device that supports the Long Range.
+### Bluetooth 5 Long Range Support - the Ability to Receive Data
 
-1. Prepare the device tested (DUT) for the Long Range receiving support by running [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp).
-2. Gather another device (Master) that passed the Elimination Criteria and run [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp). 
-	* Alternatively: gather any other device that supports Long Range advertisements (might be Remote ID add-on or Bluetooth development kit - e.g., [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)).
-3. On the Master, go to the Advertiser tab and create a New advertising packet. 
-	* In options, select Advertising Extensions and in Primary and Secondary PHY, select LE Coded (Long Range). Leave the other fields default.
-4. On the Master, run the Advertisement by toggling the switch. Select Until manually turned off and No maximum options.
+Suppose the previous test did not eliminate the device from Long Range support.
+In that case, it is now necessary to test the target device’s basic receiving capability.
+At the moment, this information cannot be read from the OS information and must be verified by using another device that supports the Long Range.
+
+1. Prepare the Device Under Test (DUT) for the Long Range receiving support by running [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp).
+  * In the Settings menu of the app, select Scanner and set the Scanning period to 5 minutes
+2. Use another device (Master) that passed the Elimination Criteria and run [nRF Connect for Mobile](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp).
+	* Alternatively: use any other device that supports Long Range advertisements (might be Remote ID add-on or Bluetooth development kit - e.g., [nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)).
+3. On the Master, go to the Advertiser tab and create a New advertising packet.
+	* In options, select Advertising Extensions and in both Primary and Secondary PHY, select LE Coded (Long Range).
+	* Press the "Add Record" and select "Complete Local Name".
+	* Set the Interval to 160 (	resulting in 100 ms).
+	* Set the Tx power to -1 (the maximum).
```

Comment:
I think you meant +1 dBm, didn't you? Or maybe your phone supports -1 as a maximum? I can set +1 for my OnePlus 8T

Reply:
Sorry, my mistake. Will fix.